mediorum/main.go

```diff
@@ -88,7 +89,7 @@ func startStagingOrProd(isProd bool) {
 	config := server.MediorumConfig{
 		Self: server.Peer{
 			Host:   httputil.RemoveTrailingSlash(strings.ToLower(creatorNodeEndpoint)),
-			Wallet: walletAddress,
+			Wallet: strings.ToLower(walletAddress),
```

Comment:
just curious... why lower this here?

Reply:
just for conformity and easier cmd+f on healthz. no particular code functionality reason